### DISCUSSION TITLE
sketch upload error

### DISCUSSION THREAD

**Discussion by akisinn** [sanni/cartreader#382] [Need help]
Hello.
An error occured when sketch uploading.
I used [V8.4_Portable.zip] and reffered this page: https://github.com/sanni/cartreader/wiki/How-to-flash-the-Arduino
I don't know the solution, please help me.
(I'm not good at English...)

---error message start---
Arduino: 1.8.16 (Windows 10), Board: "Arduino Mega or Mega 2560, ATmega2560 (Mega 2560)"
Sketch uses 227174 bytes (89%) of program storage space. Maximum is 253952 bytes.
Global variables use 6782 bytes (82%) of dynamic memory, leaving 1410 bytes for local variables. Maximum is 8192 bytes.
Low memory available, stability problems may occur.
An error occurred while uploading the sketch
avrdude: verification error, first mismatch at byte 0x0092
         0x87 != 0x97
avrdude: verification error; content mismatch
This report would have more information with
"Show verbose output during compilation"
option enabled in File -> Preferences.
---error message end---

**Comment by sanni** [comment]
Try a different PC or a different USB cable. The programming makes write errors. I have not seen this so far.
